Packs/SlackITAdmin/Integrations/Slack_IT_Admin/Slack_IT_Admin.py

```diff
@@ -0,0 +1,724 @@
+    return (
+        readable_output,
+        {},
+        res_json)
+
+
+def update_group_members_command(client, args):
+    group_id = args.get('groupId')
+    group_name = args.get('groupName')
+
+    if not (group_id or group_name):
+        return_error("You must supply either 'groupId' or 'groupName")
+    if not group_id:
+        group_id = client.get_group_id(group_name)
+
+    member_ids_to_add = args.get('memberIdsToAdd')
+    member_ids_to_delete = args.get('memberIdsToDelete')
+
+    member_ids_json_list = []
+    if member_ids_to_add:
+        if type(member_ids_to_add) is not list:
+            member_ids_to_add = json.loads(member_ids_to_add)
+        for member_id in member_ids_to_add:
+            member_ids_json_list.append(
+                {
+                    "value": member_id
+                }
+            )
+    if member_ids_to_delete:
+        if type(member_ids_to_delete) is not list:
+            member_ids_to_delete = json.loads(member_ids_to_delete)
+        for member_id in member_ids_to_delete:
+            member_ids_json_list.append(
+                {
+                    "value": member_id,
+                    "operation": "delete"
+                }
+            )
+
+    group_input = {'schemas': [SLACK_SCIM_CORE_SCHEMA_KEY], 'members': member_ids_json_list}
+
+    res = client.update_group(group_id, group_input)
+
+    if res.status_code != 204:
+        error_json = res.json()
+        return_error(f"Error in API call. Status Code: [{res.status_code}]. Error Response: {error_json}")
+
+    readable_output = f"Updated Slack Group Members for group : {group_id}"
+
+    return (
+        readable_output,
+        {},
+        None
+    )
+
+
+def replace_group_command(client, args):
+    demisto.log(str(args))
+    group_id = args.get('groupId')
+    group_name = args.get('groupName')
+
+    if not (group_id or group_name):
+        return_error("You must supply either 'groupId' or 'groupName")
+    if not group_id:
+        group_id = client.get_group_id(group_name)
+
+    group_input = {'schemas': [SLACK_SCIM_CORE_SCHEMA_KEY], 'displayName': args.get('newGroupName')}
+    members = []
+    member_ids = args.get('memberIds')
+    if member_ids:
+        if type(member_ids) is not list:
+            try:
+                member_ids = json.loads(member_ids)
+            except JSONDecodeError:
+                return_error("memberIds is not a valid list")
+            for member_id in member_ids:
+                members.append({"value": member_id})
+
+    group_input['members'] = members
+    res = client.replace_group(group_id, group_input)
+    res_json = res.json()
+
+    if res.status_code not in [200, 201]:
+        return_error(f"Error in API call. Status Code: [{res.status_code}]. Error Response: {res_json}")
+
+    readable_output = tableToMarkdown(name=f"Slack Group replaced: {group_id}",
+                                      t=res_json)
+
+    return (
+        readable_output,
+        {},
```

Comment:
please add the data to the context as well (and update in the yml the context output)